extension/agenthealth/config.go

```diff
@@ -12,6 +12,7 @@ import (
 type Config struct {
```

Comment:
IMO status code stats should have to be enabled. `agenthealth/metrics` and `agenthealth/traces` won't have any allowlisted operations, but are still going to have the handlers created and attached.

Reply:
I agree with this and I changed it so that we would have to enable statuscodes